plugin/evm/vm.go

```diff
@@ -641,7 +641,6 @@ func (vm *VM) Initialize(
 }
 
 func (vm *VM) initializeMetrics() error {
-	metrics.Enabled = true
```

Comment:
this was removed since it was causing data race conditions (duh global variable...);
It's not needed anymore since it's true by default and never disabled anywhere.